src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralParsingTests.cs

```diff
@@ -0,0 +1,1421 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
```

Comment:
nit: file-scoped while you can? (also for `RawInterpolatedStringLiteralCompilingTests`, `RawInterpolationTests` and `RawInterpolationTests`)